.nvmrc

```diff
@@ -1 +1 @@
-18.11.0
\ No newline at end of file
+20.0.0
```

Comment:
Use the exact latest 20.x.x version here.

Reply:
Use the latest version from here,
https://nodejs.org/en/about/previous-releases